packages/wrangler/src/init.ts

```diff
@@ -215,20 +215,19 @@ type WorkersDevRes = {
 	enabled: boolean;
 };
 type CronTriggersRes = {
-	schedules: [
-		{
-			cron: string;
-			created_on: Date;
-			modified_on: Date;
-		},
-	];
+	schedules: {
+		cron: string;
+		created_on: Date;
+		modified_on: Date;
+	}[];
 };
 
 function isNpm(packageManager: PackageManager) {
 	return packageManager.type === "npm";
 }
 
-async function getWorkerConfig(
+// TODO: move this to a shared location
```

Comment:
Like for `Diff` I'd prefer not to do such refactoring as part of this PR as that would significantly (and with little gain) increase the diffs people would need to review here, if it's ok I'll do it as a followup